_posts/2023-04-03-A-Step-by-Step-Guide-to-Deploying-and-Operating-an-Acoustic-Modem-running-UnetStack-to-Achieve-the-Best-Performance.md

```diff
@@ -0,0 +1,173 @@
+
+Let's say you are deploying two acoustic modems (modem A and B) in water to establish a communication link between them. For this article, we use the example of a Subnero S60H series acoustic modem. After installing and powering up both modems, you can access the modems' web interface (over the Ethernet interface) by connecting to their respective IP addresses using a web browser. Once you have logged into the modem's web interface, the landing dashboard shows the current settings along with a web shell.
+
+<p align="center" width="100%">
+   <img width="80%" src="../assets/img/landing.png"> 
+</p>
+<p align="center"><em>Landing dashboard of a Subnero S60H series modem</em></p>
+
+Depending on the distance between the devices, the first thing you may want to adjust is the power level setting of the two modems to avoid saturating the receiving modem. There is a slider in the landing page that you can use for this (default value set to -42) or you can use the `plvl` command in the web shell.
+
+>  NOTE: We recommend using the following power levels (`plvl`) as the starting values for different distances - 42 to -20 for distances under 10 meters or in air, -20 to -10 for distances between 10 and 100 meters, and -10 and above for distances over 100 meters.
+
+## The `ping` & `tell` commands
+
+Once the power levels are set, the next thing is to test the basic connectivity between the modems, before tuning them to achieve the best performance. First, note the node address of both the modems which we will be using to address each of them. In this example, modem A has a node address of 100, and modem B's address is 120.
+
+Now, we can test the connectivity between both modems using the `ping` command. You can type the following in the web shell of modem A:
+
+```
+> ping 120
+PING 120
+Response from 120: seq=0 rthops=2 time=13974 ms
+Response from 120: seq=1 rthops=2 time=12246 ms
+Response from 120: seq=2 rthops=2 time=12478 ms
+3 packets transmitted, 3 packets received, 0% packet loss
+```
+
+This shows that modem A was able to ping modem B and get the response back. It has transmitted 3 ping packets and got a response for all of them.
+
+The `ping` command uses `CONTROL` channel, which is the robust communication channel provided by UnetStack, typically used to exchange control information between the modems. 
+
+> The `CONTROL` channel is meant for low-rate robust data transmission, whereas the `DATA` channel is typically configured for higher-rate data transmission. You can read more about these in the [unet handbook](https://unetstack.net/handbook).
+ 
+However, most of the actual payload transmissions are done using the `DATA` channel. So, our next step is to verify that the `DATA` channel works fine between the modems. This can be achieved using the `tell` command. Type the following command in the web shell of modem A.
```

Comment:
Maybe we can also note that `tell` uses a combination of both `CONTROL` and `DATA` channel?